### PR TITLE
Improve desktop experience

### PR DESCRIPTION
**Pull request by liudonghua123**
In windows desktop, the physicalKey do not match the actural key pressed, use logicalKey instead. And the upper right donation button does not lose focus when press-release (maybe it's a bug), so manually unfocus it.

And update the vendor files (android/ios/web/macos/linux/windows) via `flutter create  --project-name flutter_tetris --org tech.soit --overwrite .`

### CODE REVIEW

File: lib/panel/page_land.dart [hunk 1]
```diff
@@ -40,6 +40,11 @@ class PageLand extends StatelessWidget {
                                   context: context,
                                   builder: (context) => DonationDialog());
                             },
+                            onHighlightChanged: (pressed) {
+                              if(!pressed) {
```
Comment (boyan01) [inline lib/panel/page_land.dart]: Maybe should insert a space after `if` key words

File: lib/panel/page_portrait.dart [hunk 2]
```diff
@@ -29,6 +29,11 @@ class PagePortrait extends StatelessWidget {
                             context: context,
                             builder: (context) => DonationDialog());
                       },
+                      onHighlightChanged: (pressed) {
+                        if(!pressed) {
```
Comment (boyan01) [inline lib/panel/page_portrait.dart]: the same as front

**Comment by boyan01** [comment]
Good Job. Thanks very much for your contribution.

But it seems has a code format issues, if you can fix them, it will be excellent.

**Review by boyan01 — CHANGES_REQUESTED** [review]
code format.

**Comment by liudonghua123** [comment]
@boyan01 Hi, I just updated the pr, now the modified code is formatted.

**Review by boyan01 — APPROVED** [review]

**Comment by boyan01** [comment]
LGTM , thanks for your contribution.

**Pull request merged**